### PR TITLE
Refactor data dictionary transaction isolation setting

### PR DESCRIPTION
**Pull request by laurynas-biveinis**
InnoDB uses READ UNCOMMITTED, which is not supported with MyRocks. Instead of
hardcoding READ UNCOMMITTED at several locations, introduce a helper function
that returns the desired DD transaction isolation level, based on the default DD
engine.

No functional changes if the default DD engine is InnoDB.

### CODE REVIEW

**Comment by laurynas-biveinis** [comment]
There are two commits but only the second one is for review. The first one will be removed once the prerequisite branch is merged.

File: storage/rocksdb/get_rocksdb_files.sh [hunk 1]
```diff
@@ -1,4 +1,8 @@
 #!/bin/bash
+if [ -z "$PATH" ]; then
+    export PATH="/sbin:/usr/sbin:/bin:/usr/bin"
```
Comment (luqun) [inline storage/rocksdb/get_rocksdb_files.sh:3]: unrelated changes?

Comment (laurynas-biveinis) [reply storage/rocksdb/get_rocksdb_files.sh:3]: This branch contains a commit that fixes the build for me. I'll remove it once it is merged, until then, please review 3dae3ef

File: sql/dd/impl/tables/dd_properties.cc [hunk 2]
```diff
@@ -117,12 +118,7 @@ bool DD_properties::init_cached_properties(THD *thd) {
   // Early exit in case the properties are already initialized.
   if (!m_properties.empty()) return false;
 
-  /*
-    Start a DD transaction to get the properties. Please note that we
-    must do this read using isolation level ISO_READ_UNCOMMITTED
```
Comment (luqun) [inline sql/dd/impl/tables/dd_properties.cc:122]: could we keep these comments? maybe useful for myrock atomic DDL

Comment (laurynas-biveinis) [reply sql/dd/impl/tables/dd_properties.cc:122]: They are consolidated in dd_utility.h
